src/node_report.cc

```diff
@@ -70,6 +68,7 @@ const char* v8_states[] = {"JS", "GC", "COMPILER", "OTHER", "EXTERNAL", "IDLE"};
 static bool report_active = false; // recursion protection
 static char report_filename[NR_MAXNAME + 1] = "";
 static char report_directory[NR_MAXPATH + 1] = ""; // defaults to current working directory
+static std::string versionString = "Unable to determine Node.js version\n";
```

Comment:
Can you use snake_case for consistency?  (Here and elsewhere, e.g., also please update processProp, versionProp, and so on.)